### YamlMime:PythonClass
uid: azure.cosmos.exceptions.CosmosResourceNotFoundError
name: CosmosResourceNotFoundError
fullName: azure.cosmos.exceptions.CosmosResourceNotFoundError
module: azure.cosmos.exceptions
inheritances:
- azure.core.exceptions.ResourceNotFoundError
- azure.cosmos.exceptions.CosmosHttpResponseError
summary: An HTTP error response with status code 404.
constructor:
  syntax: CosmosResourceNotFoundError(status_code=None, message=None, response=None,
    **kwargs)
  parameters:
  - name: status_code
    description: HTTP response code.
    defaultValue: None
    types:
    - <xref:int>
  - name: message
    description: Error message.
    defaultValue: None
    types:
    - <xref:str>
  - name: response
    defaultValue: None
methods:
- uid: azure.cosmos.exceptions.CosmosResourceNotFoundError.add_note
  name: add_note
  summary: "Exception.add_note(note) \u2013\nadd a note to the exception"
  signature: add_note()
- uid: azure.cosmos.exceptions.CosmosResourceNotFoundError.raise_with_traceback
  name: raise_with_traceback
  summary: 'Raise the exception with the existing traceback.


    *Deprecated since version 1.22.0:* This method is deprecated as we don''t support
    Python 2 anymore. Use raise/from instead.'
  signature: raise_with_traceback() -> None
- uid: azure.cosmos.exceptions.CosmosResourceNotFoundError.with_traceback
  name: with_traceback
  summary: "Exception.with_traceback(tb) \u2013\nset self.__traceback__ to tb and\
    \ return self."
  signature: with_traceback()
attributes:
- uid: azure.cosmos.exceptions.CosmosResourceNotFoundError.args
  name: args
- uid: azure.cosmos.exceptions.CosmosResourceNotFoundError.continuation_token
  name: continuation_token
  signature: 'continuation_token: str | None'
- uid: azure.cosmos.exceptions.CosmosResourceNotFoundError.error
  name: error
  signature: 'error: ODataV4Format | None'
- uid: azure.cosmos.exceptions.CosmosResourceNotFoundError.exc_msg
  name: exc_msg
  signature: 'exc_msg: str'
- uid: azure.cosmos.exceptions.CosmosResourceNotFoundError.exc_traceback
  name: exc_traceback
  signature: 'exc_traceback: TracebackType | None'
- uid: azure.cosmos.exceptions.CosmosResourceNotFoundError.exc_type
  name: exc_type
  signature: 'exc_type: Type[Any] | None'
- uid: azure.cosmos.exceptions.CosmosResourceNotFoundError.exc_value
  name: exc_value
  signature: 'exc_value: BaseException | None'
- uid: azure.cosmos.exceptions.CosmosResourceNotFoundError.inner_exception
  name: inner_exception
  signature: 'inner_exception: BaseException | None'
- uid: azure.cosmos.exceptions.CosmosResourceNotFoundError.message
  name: message
  signature: 'message: str'
- uid: azure.cosmos.exceptions.CosmosResourceNotFoundError.model
  name: model
  signature: 'model: Any | None'
- uid: azure.cosmos.exceptions.CosmosResourceNotFoundError.reason
  name: reason
  signature: 'reason: str | None'
- uid: azure.cosmos.exceptions.CosmosResourceNotFoundError.response
  name: response
  signature: 'response: _HttpResponseCommonAPI | None'
- uid: azure.cosmos.exceptions.CosmosResourceNotFoundError.status_code
  name: status_code
  signature: 'status_code: int | None'
